apps/docs/components/Navigation/NavigationMenu/TopNavBar.tsx

```diff
@@ -118,6 +87,11 @@ const TopNavBar: FC = () => {
               </a>
             </Button>
           )}
+          {process.env.NEXT_PUBLIC_DEV_AUTH_PAGE === 'true' && (
```

Comment:
Needed to test auth on local dev. The proxy magic that makes signin via dashboard work on prod and (with some manual trickery) on preview doesn't work on local dev.

I tried proxying everything through www served on localhost and ended up with some mad infinite reloading from Next.js Fast Refresh. Couldn't for the life of me figure out why after banging my head against it for hours